### DISCUSSION TITLE
Quick Launch Toolbar [AND/OR] System Tray

### DISCUSSION THREAD

**Discussion by hschlossberg** [valinet/ExplorerPatcher#3180] [General]
I've been using ExplorerPatcher ever since Win 10 came out, and I've now been in Win 11 for at least a year with it.  My setup has been that I've always had both the Quick Launch toolbar **_AND_** the System Tray displayed on my taskbar.  Now it seems I only get to have one _**OR**_ the other.

Two days ago, Windows forced an update in the middle of the night and life has been hell ever since. 
> Edition	Windows 11 Pro
> Version	23H2
> Installed on	‎4/‎23/‎2024
> OS build	22631.3447

I've read elsewhere that MS changed the taskbar in a way that drastically changes the structure behind the system tray, but it seems ExplorerPatcher IS able to partly accommodate that.  If I set my taskbar style to Windows 11, I get the system tray but not the QuickLaunch.  If I set the taskbar style to Windows 10, I get the opposite.  (See screenshots below.) Is there no way to still get both?
![image](https://github.com/valinet/ExplorerPatcher/assets/5552927/853ce10e-129f-4a7b-af06-ae0c464eb185)
![image](https://github.com/valinet/ExplorerPatcher/assets/5552927/03d8c112-41ab-4961-a338-3f0155c0ec5e)

(FWIW, I prefer the Win 10 style taskbar, but if I had to make a choice I need the systray more than the QL bar.)

**Comment by pyrates999** [comment]
I read on another post that Microsoft removed the system tray from the windows 10 taskbar.

**Reply by hschlossberg** [reply]
I thought it was from Win 11 that they removed the systray. Either way, I guess I was thinking if EP figured out how to display the systray for one "style' then why not for the other style?  I had it until earlier this week using Win 11 version 22H2 with EP's Windows 10 menu style.  But not now using 23H2.  Sorry, I guess I'm confused about what's what on these changes and on how EP implements things.

**Reply by pyrates999** [reply]
This is it: https://github.com/valinet/ExplorerPatcher/discussions/2363

Looks like Microsoft released the code to hide the system tray on the windows 10 taskbar in the April 9 2024 update.
